include/deviceconfig.h

```diff
@@ -338,6 +356,34 @@ class DeviceConfig : public IJSONSerializable
         SetAndSave(rememberCurrentEffect, newRememberCurrentEffect);
     }
 
+    uint8_t GetBrightness() const
+    {
+        return brightness;
+    }
+
+    ValidateResponse ValidateBrightness(const String& newBrightness)
+    {
+        auto newNumericBrightness = newBrightness.toInt();
+
+        if (newNumericBrightness < BRIGHTNESS_MIN)
+            return { false, String("brightness is below minimum value of ") + BRIGHTNESS_MIN };
+
+        if (newNumericBrightness > BRIGHTNESS_MAX)
+            return { false, String("brightness is above maximum value of ") + BRIGHTNESS_MAX };
+
+        return { true, "" };
+    }
+
+    void SetBrightness(int newBrightness)

```

Comment:
While the brightness is actually a unit8_t, this function accepts an int. The main benefit of that is that we are friendlier towards callers, and we sort out the matter of keeping the actual brightness within defined boundaries. An additional benefit is that we don't have to add another specialization for CWebServer::PushPostParamIfPresent (and if we're pedantic, a new data type to SettingSpec).